raiden/tests/integration/long_running/flaky/test_token_networks.py

```diff
@@ -51,57 +92,70 @@ def test_participant_selection(raiden_network, token_addresses):
     gevent.wait(connect_greenlets)
 
     # wait some blocks to let the network connect
-    wait_blocks = 15
-    for _ in range(wait_blocks):
-        for app in raiden_network:
-            wait_until_block(
-                app.raiden.chain,
-                app.raiden.chain.block_number() + 1,
-            )
+    for app in raiden_network:
+        wait_until_block(
+            app.raiden.chain,
+            app.raiden.chain.block_number() + 1,
```

Comment:
Why decrease the number of waiting blocks? p.s.: I don't know why it was 15 blocks previously.

Reply:
I have no idea either. I've removed the whole block and it still works....